authui/package-lock.json

```diff
@@ -19,7 +19,7 @@
         "axios": "1.1.3",
         "cropperjs": "1.5.12",
         "ethers": "5.5.1",
-        "intl-tel-input": "17.0.19",
+        "intl-tel-input": "github:tw-360vier/intl-tel-input#d8a0b82f22366b6caea0e9ffebc7e20e6b1dd145",
```

Comment:
One concern is if the owner of PR rebase the branch, the commit hash will become invalid. I suggest we fork this repository and the branch under https://github.com/authgear and use our own fork.

Reply:
Updated to use a authgear fork